### ISSUE TITLE
Bot no longer using server-specific Discord nicknames, using default Discord usernames instead

### ISSUE THREAD

**Issue by meowxguy** [reactiflux/discord-irc#569]
Been running this bot for a few years now and just recently it started using default Discord usernames in IRC instead of the name specific to the Discord server. This also means that the webhook avatar icons don't match up when typing in IRC and it posting back to Discord. Haven't made any configuration changes to the bot.

https://i.imgur.com/eq1D6Fg.png
https://i.imgur.com/b2jt5BF.png

**Comment by Throne3d** [comment]
Hey there! What version of discord-irc are you on? Additionally, do you receive any sort of error in the console when this happens? I just tested it on v2.9.0 on a local server, and it seems to track my nickname changes fine between "Throne" and "ABC", so this doesn't look straightforward to resolve from my end.

**Comment by meowxguy** [comment]
hm, 2.6.1 no, no errors are logged.

will give updating a shot.

**Comment by meowxguy** [comment]
updating has fixed the problem. who would have thought?

**Comment by Throne3d** [comment]
Awesome! Discord probably changed something in the structure of the API response, and the old version of discord.js doesn't recognize it.

I'll investigate adding this to the README – I have a few other items I could add there in case anyone else encounters the same priblem!.

**Comment by jmbjr** [comment]
hm. updating to 2.9.0 didn't change my problem. the bot is always displaying the underlying account name in irc

**Comment by jmbjr** [comment]
here's a screenshot. my account is @johnstein but I changed it to juanpiedra for this server.  in discord, my name is 'juanpiedra' but when I type anything, the log from the bot shows <johnstein>, my account name. 
![image](https://user-images.githubusercontent.com/3990931/100529431-9175ae80-319c-11eb-9586-2fdc9bb67565.png)
I'm on 2.9.0 using commit 4f449954e04419b24b6baf2520ad38dce59d785f and I've closed and restarted the bot several times.
